### ISSUE TITLE
Load from DB problem

### ISSUE THREAD

**Issue by juju95** [djpate/jTag#11]
When  i try to load tags from DB i always get a "Wrong Object" of the returned list
And on the second one and next are shown

No error in the json returned, and if i random the tags order in the list, it is always the 1st row who return this error

And if i copy past my returned json in the defaultags, they are working perfectly 
any idea

**Comment by antcptr** [comment]
This could happen when no 'jTagTag' classes are found. Could try calling the list action with some delay.
Like this:

` setTimeout(function(){ 
 $.getJSON("ajax.php",{'action':'list'},function(tags){
                $.each(tags, function(key,tag){
                $("#img").addTag(tag.width,tag.height,tag.top,tag.left,tag.label,tag.id);
                });
                });

}, 1000);`

or
call list action after img load

`
$('#img').load(function(){
                $.getJSON("ajax.php",{'action':'list'},function(tags){
                $.each(tags, function(key,tag){
                $("#img").addTag(tag.width,tag.height,tag.top,tag.left,tag.label,tag.id);
                });
        });
});
`
